build

```diff
@@ -3,14 +3,29 @@
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/etcd"
 
+GIT_SHA=`git rev-parse --short HEAD || echo "GitNotFound"`
+
+LINK_OPERATOR="="
+
+: ${ARCH_LIST:="amd64 arm64"}
```

Comment:
@heyitsanthony I think its a good idea to build all supported arches by default.  If a user only wants to build a specific set they can set those with ARCH_LIST.

With your suggestion to build all arches the user needs to invoke the build script multiple times.